client/pages/groups.jsx

```diff
@@ -228,7 +336,15 @@ export default class Groups extends React.Component {
     }
   }
 
+  componentDidUpdate() {
+    localStorage.setItem('editing-state', JSON.stringify(this.state.isEditing));
+    localStorage.setItem('brackets-state', JSON.stringify(this.state.brackets));
+    localStorage.setItem('groupStage-state', JSON.stringify(this.state.groupStage));
```

Comment:
Sticking this stuff in `localStorage` is a little weird. Why is this necessary?

Reply:
After consulting with Scott, I decided to put it in there, since the group stage picks (and the knockout round picks) are a multi step process, to be able to save the users selections incase of internet failure or sudden page refresh, they do not lose their selections and have to restart (they don't need to 'save' or 'send a fetch post request' after every single selection. Instead once they are done with one set (group or knockout selections) then they 'save' and 'send a post request' to send to the database.